### ISSUE TITLE
Verify list of toll-free phone numbers from IRIS for LC MVP

### ISSUE THREAD

**Issue by bethpottsVADEPO** [department-of-veterans-affairs/va.gov-team#13888]
## Issue Description
We need to make sure all the phone numbers we'll list for Veterans are up-to-date.

---
## Tasks
- [x] Call the phone numbers on this IRIS page (linked to below) to verify 1/ they go through and get picked up by an automated system and 2/ they get picked up by the correct automated system (in other words, if we list a number for health care benefits, it goes to the appropriate place rather than, say, a life insurance line). (Selina)

**Note: Do not call the Crisis Line. We know that one is correct and we don't want to clutter their line!**

https://iris.custhelp.va.gov/app/answers/detail/a_id/1703/kw/Toll%20Free%20Numbers%20for%20Contacting%20VA

## Acceptance Criteria
- [x] Numbers all confirmed and if there are any incorrect they are noted in this ticket.
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by RLHecht** [comment]
@lunascoop Here's the ticket I just discussed with you, Please let Beth or I know if you have any questions. Thanks!

**Comment by bethpottsVADEPO** [comment]
@lunascoop Thank you for doing this! And thank you for **not** calling the crisis line.

**Comment by lunascoop** [comment]
I have called the numbers listed on the linked page. Here are my notes:

- Combat call center, has changed its name to Vets Center Call Center (for trauma such as PTSD, etc.)
- There are two numbers listed to handle the Children of Women Vietnam Veterans (CWVV), the Foreign Medical Program (FMP) and the Spina Bifida Health Care Program. The first number, 1-877-345-8179, seems to be specifically for FMP, and the second, 1-888-820-1756, for the Spina Bifida Health Care Program. I checked out the [Children of Women Vietnam Veterans (CWVV)](https://www.va.gov/COMMUNITYCARE/programs/dependents/cwvv/index.asp) page, which does list 1-888-820-1756 as the contact for general info. But again, the automated system only references the Spina Bifida Health Care Program and gives the caller the option to say "Claims, Eligibility or Preauthorization." When I called the number listed for Eligibility Application, 303-914-2900, it just kept ringing and no one picked up. 
- The Mammography Helpline number is no longer in service. I checked out the [Understanding your Mamogram](https://www.womenshealth.va.gov/WOMENSHEALTH/OutreachMaterials/ReproductiveHealth/UnderstandingYourMammogram.asp) page and it lists the Women Veterans Call Center as a resource (which is called the Women Veterans Hotline on the toll-free number page).
